docs/server.md

```diff
@@ -67,6 +67,12 @@ Defaults to `false`.
 
 A valid postgres URI, where the user and history data will be saved to.
 
+### path
+
+A path to prepend to all the routes of the server.
+
+Defaults to `"/"`
```

Comment:
I wonder if we should default to "" and clarify that means no prefix. And since the prefix doesn't need a trailing `/`, I think this would save some potential confusion

Reply:
Sure, the reason I did it was that nest() panics if you use an empty route